src/pages/Onboarding/Onboarding.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="onBoarding">
+    <div class="wrapper">
+      <header>
+        <img src="./../../assets/img/header-logo.svg" class="header-logo">
+      </header>
+      <main>
+        <section id="welcome">
+          <div class="cube-div">
+            <img src="./../../assets/img/onboarding/identity-card.svg" class="large-copy-2">
+          </div>
+          <div class="content">
+            <h1>Welcome to Identity Manager</h1>
+            <p>
+              æternity gives you easy access to the magic of blockchain technology.
+              Follow these quick steps or just jump in.
+            </p>
+          </div>
+        </section>
+        <section id="manage">
+          <div class="screenshot-phone">
+            <div class="image">
+              <img src="./../../assets/img/onboarding/mini-card.svg" class="Group">
+              <img src="./../../assets/img/onboarding/mini-card1.svg" class="Group">
+              <img src="./../../assets/img/onboarding/mini-card2.svg" class="Group">
+              <img src="./../../assets/img/onboarding/mini-card3.svg" class="Group">
+              <img src="./../../assets/img/onboarding/mini-card4.png" srcset="./../../assets/img/onboarding/mini-card4@2x.png 2x, ./../../assets/img/onboarding/mini-card4@3x.png 3x" class="Group-4">
```

Comment:
Seems that you don't use classes `Group` and `Group-4` anywhere. Better to remove this classes.